packages/odyssey-react/src/components/Status/Status.module.scss

```diff
@@ -14,12 +14,13 @@ $status-indicator-size: 1em * 0.5;
 
 .label {
   display: block;
-  margin-block-end: $spacing-xs;
+  margin-block-end: var(--SpaceFixedXs);
   font-weight: 600;
 }
 
 .value {
   position: relative;
+  // padding-inline-start: calc(#{$status-indicator-size + var(--SpaceRelativeXs));
```

Comment:
@jeffbelser-okta here's an example of where I'm trying to use `calc()`. Since we're using `var()`, we'll need to replace our Sass math with `calc()` wherever it appears.

Oddly enough, the same babel transform error occurs with this as a comment as well.

Reply:
@jeffbelser-okta to keep this moving, are you cool if I leave anything that requires `calc()` in Sass variables until we find a fix?

Reply:
@edburyenegren-okta what progress have you made here debugging? Any new information or approaches to share?

If the plan is to leave these as compile time only SASS variables we should document them in new JIRA's so that we don't loose track of the required work to be done.

Reply:
afaict we'll need to refactor the theme reducer to get this working. `calc()` + `var()` works in plain CSS and Sass, even deeply nested ones or vars with fallbacks. our babel transform, however, chokes somewhere along the way.

I wasn't able to get further than that amount of testing, and the error isn't super helpful :(

For now, I've created this ticket to capture all instances as I continue the theme refactor work: https://oktainc.atlassian.net/browse/OKTA-455370